Test/Fixture/IcingVersionFixture.php

```diff
@@ -19,11 +19,20 @@ class IcingVersionFixture extends CakeTestFixture {
 		'created' => array('type' => 'datetime', 'null' => true, 'default' => null),
 		'url' => array('type' => 'string', 'null' => true, 'default' => null, 'collate' => 'utf8_general_ci', 'charset' => 'utf8'),
 		'ip' => array('type' => 'string', 'null' => true, 'default' => null, 'length' => 50, 'collate' => 'utf8_general_ci', 'charset' => 'utf8'),
-		'is_delete' => array('type' => 'boolean', 'null' => false, 'default' => null),
+		'is_soft_delete' => array('type' => 'boolean', 'null' => false, 'default' => null),
+		'is_delete' => array('type' => 'boolean', 'null' => false, 'default' => '0', 'key' => 'index', 'comment' => 'true if the model was deleted.'),
+		'is_soft_delete' => array('type' => 'boolean', 'null' => false, 'default' => '0', 'key' => 'index', 'comment' => 'true if version is soft deleted.'),
```

Comment:
You've got duplicate is_soft_delete's here, not sure if it was supposed to be is_minor_version or not

Reply:
Good catch.